pennylane_lightning/core/src/simulators/lightning_gpu/MPIWorker.hpp

```diff
@@ -175,8 +175,11 @@ make_shared_mpi_worker(custatevecHandle_t handle, MPIManager &mpi_manager,
                                             communicatorType, nullptr);
     if (err != CUSTATEVEC_STATUS_SUCCESS) {
         communicator = nullptr;
-        PL_CUSTATEVEC_IS_SUCCESS(custatevecCommunicatorCreate(
-            handle, &communicator, communicatorType, "libmpi.so"));
+        auto py_err = custatevecCommunicatorCreate(
+            handle, &communicator, communicatorType, "libmpi.so");
```

Comment:
Can we add a comment about when will this happen? Is it when MPICH/OpenMPI isn't automatically found by the communicatorcreate, so will try to explicitly search for `libmpi.so` in `LD_LIBRARY_PATH`

Reply:
correct!